selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/RenameOrganizationTest.java

```diff
@@ -74,81 +72,69 @@ public void setUp() throws Exception {
 
   @AfterClass
   public void tearDown() throws Exception {
-    testOrganizationServiceClient.deleteById(parentOrganization.getId());
+    for (OrganizationDto organization : testOrganizationServiceClient.getAll())
+      testOrganizationServiceClient.deleteById(organization.getId());
```

Comment:
Such code can affect organizations which are created in parallel threads of test execution.